report_account_move_settlement/reports/report_settlement_templates.xml

```diff
@@ -53,13 +53,22 @@
                                 t-field="o.partner_id"
                                 t-options='{"widget": "contact", "fields": ["name"], "no_marker": True}'
                             />
+                            <div t-if="o.partner_id.vat" class="mb-2">
+                                <span>登録番号:</span>
+                                <span t-field="o.partner_id.vat" />
+                            </div>
                         </div>
                     </div>
                     <div class="col-5">
                         <address
                             t-field="company.partner_id"
                             t-options='{"widget": "contact", "fields": ["name", "address"], "no_marker": True}'
+                            style="margin-bottom: 0;"
```

Comment:
If` style="margin-bottom: 0;" `is not added.
There will be a line of spaces in the middle of the "company name" and "Invoice Registration Number".

Reply:
@liuhehe1995 Please add a note in the code, not in the PR.